runtime/doc/options.txt

```diff
@@ -5862,10 +5862,14 @@ A jump table for the options with a short description can be found at |Q_op|.
 'spelloptions' 'spo'	string	(default "")
 			local to buffer
 	A comma-separated list of options for spell checking:
-	   camel	When a word is CamelCased, assume "Cased" is a
+	camel		When a word is CamelCased, assume "Cased" is a
 			separate word: every upper-case character in a word
 			that comes after a lower case character indicates the
 			start of a new word.
+	noplainbuffer	Only spellcheck a buffer when 'syntax' is enabled, or
+			or when extmarks are set within the buffer. Only
```

Comment:
extmarks with `spell = true`.

Reply:
I think that the help here should be higher level.